themes/powerline-multiline/powerline-multiline.theme.bash

```diff
@@ -74,6 +74,9 @@ THEME_CLOCK_FORMAT=${THEME_CLOCK_FORMAT:="%H:%M:%S"}
 IN_VIM_THEME_PROMPT_COLOR=${POWERLINE_IN_VIM_COLOR:=245}
 IN_VIM_THEME_PROMPT_TEXT=${POWERLINE_IN_VIM_TEXT:="vim"}
 
+IN_TOOLBOX_THEME_PROMPT_COLOR=${POWERLINE_IN_TOOLBOX_COLOR:=125}
+IN_TOOLBOX_THEME_PROMPT_TEXT=${POWERLINE_IN_TOOLBOX_TEXT:="❲T❳ "}
```

Comment:
(I'll only mention this once here since this is the first occurrence)

The powerline terraform prompt segment already uses `"❲t❳ "` in all 4 main powerline themes.

Not sure its an issue since the potential for overlap may be small and the user can customize either icon (or both!)  if they wish.

Reply:
Hmm, didn't realize the terraform one. What do you think the better text should be?